USER_GUIDE.md

```diff
@@ -31,23 +37,26 @@ For App Engine standard, the plugin exposes the following goals :
 
 #### Local Run
 
-| Goal    | Description |
-| ------- | ----------- |
-| `run`   | Run the application locally. |
-| `start` | Start the application in the background. |
-| `stop`  | Stop a running application. |
+| Goal              | Description |
+| ----------------- | ----------- |
+| `run`             | Run the application locally. |
+| `start`           | Start the application in the background. |
+| `stop`            | Stop a running application. |
+| `genRepoInfoFile` | Generates repository information files for the Stackdriver Debugger. |
```

Comment:
This doesn't seem to be in the app-gradle-plugin USER_GUIDE. Is this correct?

Reply:
yeah that's because it's a separate plugin. I'm not sure we should include this at all.